mesos_master/tests/common.py

```diff
@@ -0,0 +1,16 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import os
+
+from datadog_checks.dev import get_docker_hostname
+
+HOST = get_docker_hostname()
+
+HERE = os.path.dirname(os.path.abspath(__file__))
```

Comment:
I think this is a prime candidate to use the get_here from datadog_checks_dev - https://github.com/DataDog/integrations-core/pull/3135